text/0000-ready-for-stabilization-announcements.md

```diff
@@ -0,0 +1,92 @@
+- Feature Name: ready_for_stabilization_announcements
+- Start Date: 2018-08-26
+- RFC PR:
+- Rust Issue:
+
+# Summary
+[summary]: #summary
+
+Announce unstable features close to stabilization or needing community attention
+through TWiR.
+
+# Motivation
+[motivation]: #motivation
+
+As Rust as a language matures, it is no longer necessary to use nightly features
+for a day to day development. This results in less experimentation with features
+before they stabilize and in noticeable lack of feedback (or feedback that
+arrives after stabilization instead of before).
+
+Actively seeking out nightly features has the downside one doesn't know to which
+ones to pay attention, as there are too many and it is better to play with the
+ready ones.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+TWiR (This Week in Rust) shall announce tracking issues which are currently in
+final comment period, just like with RFC PRs in FCP.
+
+In addition, it's also possible to announce tracking issues of features where
+increased attention and input from community is desirable.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+The TWiR newsletter already allows inclusion of „key PRs“ in the „Final Comment
+Period“ section.
+
+This RFC asks for inclusion of tracking issues with visible effects in the
+„Final Comment Period“ section, and allows also including tracking issues where
+there's an explicit call for experimentation and feedback.
+
+Furthermore, it proposes to visually distinguish RFCs from tracking issues and
+to explicitly call for experimenting with the features and providing feedback.
+
+When no or very little feedback arrives on a feature, it might be a signal to
+either wait longer or consider if the feature is desired by users at all.
```

Comment:
I don't know how much that constitutes a change in policy. I meant it more like soft suggestion, that lack of feedback is kind of feedback too, but I'm not really sure how soft I'd like that suggestion to be (as mentioned in the unresolved questions).

I know some go without the feedback, but:
* I hope announcing them would bring them more attention and even the small and uncontroversial ones could get some kind of „looks good“ feedback.
* I feel there's a space in between of „desired“ and „undesired“ ‒ something like „not cared about either way“. The lack of feedback might signal that the feature is neither harmful nor beneficial in the eyes of people ‒ but it probably depends on the feature itself how to understand that.

Reply:
Here's an example of a standard library addition that hasn't had much discussion in the FCP for stabilization:
+ https://github.com/rust-lang/rust/issues/48169 , everyone who has reacted save for me and scott are lib team members..
+ https://github.com/rust-lang/rust/pull/48110 , this one had even fewer reactions...

Generally speaking for an RFC, the "big debate" (and excitement / not...) has already been had so it is not surprising that there is less fuss in the stabilization issues. Even with more exposure, I think small features will still not be talked about much in the stabilization issues. I don't think the teams should evaluate "not much response" as "undesired" and thus it should cause no extra delay (we should take care to not make the process more bureaucratic than as is...).

Reply:
Again, not „undesired“, but „not cared about“. I wouldn't say that not much feedback would necessarily mean „don't pass it“. But in case the feature would be something high-profile (for example the `async fn`), I would be surprised if it didn't receive some feedback from the community. If it did not, it would look suspicious ‒ not a signal to close it, but maybe to ask why.

Or maybe in other words, I could say that receiving significantly less feedback that could be expected, depending on the size and popularity of the feature, should hint at some need for investigation if something is amiss. But maybe that's kind of obvious thing ‒ if the effect of announcing the FCPs would be most features get some feedback, than not receiving it would look out of ordinary without any policy.

Reply:
Truthfully, I don't think no feedback happens on large features, so I don't think it is necessary to state it. I would interpret little feedback on a large feature that has already had feedback on the RFC as *"ok, nothing interesting happened since the RFC, proceeding as planned."*.